### ISSUE TITLE
Correcciones

### ISSUE THREAD

**Issue by mnmallea** [Gina04/monopoly#1]
- [ ] mapTactica (expresividad) -> debería llamarse cambiarTactica por lo que hace

- [ ] La función aumentarDinero realmente no es necesaria. Porque fijate que en realidad lo unico que hace es sumar 2 Int y devolver el resultado. Es decir, sumarDinero es simplemente la suma! Tranquilamente se puede usar + en vez de eso.

- [ ] agregarAcciones es un caso parecido al anterior

- [ ] En general hay un par de sobredelegaciones. No es necesario ponerle otros nombres a funciones que hacen cosas básicas como sumar numeros, agregar elementos a una lista. (`aumentarDinero`, `agregarAcciones`, `agregarPalabra`)

- [ ] Qué pasa en tu implementación de ultimaRonda si el participante no tuviera acciones?

**Comment by Gina04** [comment]
Hola como va, consulta por el último issue, si el participante no tuviera acciones. Eso me devuelve <function> esto está mal? deberia usar un fold con semilla?

**Comment by Gina04** [comment]
* me devuelve <function>

**Comment by Gina04** [comment]
*me devuelve una función

**Comment by mnmallea** [comment]
Exacto, si el participante no tuviera acciones, entonces va a romper. Porque tanto foldl1 como foldr1 rompen si la lista que le pasas está vacía.
Entonces podríamos usar como semilla una función que simplemente devuelva al mismo participante sin cambios (`id`)

**Comment by Gina04** [comment]
Me quedaría asi: 
ultimaRonda:: Persona->Accion
ultimaRonda unJugador= foldl (.) id (acciones unJugador)

pero cuando pruebo esto en la consola me  devuelve una función, y eso me debería devolver un jugador modificado no?

**Comment by mnmallea** [comment]
está bien que te devuelva una función porque una Accion es una funcion que toma un Jugador y devuelve un Jugador
